tests/controller_tests/test_filters.py

```diff
@@ -89,6 +89,51 @@ def test_get_funds(seeded_test_client):
     assert all(isinstance(fund["id"], str) for fund in response_json)
 
 
+class MockProgramme:
```

Comment:
is this redundant? Look like it can be removed

Reply:
done